dos_connect/server/memory_backend.py

```diff
@@ -101,6 +101,8 @@ def search(properties, index='data_objects', size=DEFAULT_PAGE_SIZE,
     if include_total set, return tuple (hit, total)
     """
     hits = []
+    if index not in store:
+        store[index] = []
```

Comment:
This is fine, but just in case you don't know about it: https://docs.python.org/2/library/collections.html#collections.defaultdict